framework/test/unit/abi_handler/abi_handler.spec.ts

```diff
@@ -169,6 +170,36 @@ describe('abi handler', () => {
 		});
 	});
 
+	describe('cacheGenesisState', () => {
```

Comment:
Missing test for checking if `context.inner.stateStore` gets closed. Same for invoke test

Reply:
in the `cacheGenesisState` it was created and close in the same function, so it was hard to spy on them 😢